code/mission/missionmessage.cpp

```diff
@@ -2121,15 +2126,23 @@ int get_builtin_message(int type, int persona, ship* sender, ship* subject, bool
 	} else {
 		int fallback = Builtin_messages[type].fallback;
 		if (fallback != MESSAGE_NONE) {
-			return get_builtin_message(fallback, persona, sender, subject, true);
-		} else if (require_exact_persona_match && persona_allows_substitution(persona)) {
```

Comment:
This is the only place where `persona_allows_substitution` is referenced.  Should it be added somewhere else?

Reply:
Yes, thank you for catching that.